### PR TITLE
fixes bug 1164512 - Creating API tokens manually not recorded as admin events

### PR DESCRIPTION
**Pull request by peterbe**
@rhelmer r?

### CODE REVIEW

**Comment by peterbe** [comment]
Or @AdrianGaudebert r? if you have time? Rob might be too busy.

**Comment by AdrianGaudebert** [comment]
r+

**Comment by peterbe** [comment]
@jdotpz In fact, whenever you have that memcache thing ready and want to try a new automated stage push feel free to merge this bad boy.

File: webapp-django/crashstats/manage/views.py [hunk 1]
```diff
@@ -831,6 +832,18 @@ def api_tokens(request):
             for permission in data['permissions']:
                 token.permissions.add(permission)
 
+            log(request.user, 'api_token.create', {
+                'user': token.user.email,
+                'expires': token.expires,
+                # Do this reverse trick to avoid microseconds rounding it
```
Comment (rhelmer) [inline webapp-django/crashstats/manage/views.py]: this one weird trick

**Comment by jdotpz** [comment]
pushing it now, thanks!  memcached should be working properly

**Comment by peterbe** [comment]
"Everything looks good!"
https://crash-stats.mocotoolsstaging.net/_debug_login/

**Pull request merged**